sim/network.go

```diff
@@ -61,24 +61,68 @@ func (n *Network) AddParticipant(p gpbft.Receiver) {
 	n.participants[p.ID()] = p
 }
 
+////// Network interface
+
+func (n *Network) NetworkFor(signer gpbft.Signer, id gpbft.ActorID) *networkFor {
+	return &networkFor{
+		ParticipantID: id,
+		Signer:        signer,
+		Network:       n,
+	}
+}
+
+type networkFor struct {
+	ParticipantID gpbft.ActorID
+	Signer        gpbft.Signer
+	*Network
+}
+
+// Log implements tagging Tracer interface
+func (nf *networkFor) Log(format string, args ...interface{}) {
```

Comment:
```suggestion
func (nf *networkFor) Log(format string, args ...any) {
```

Reply:
@adlrocha how come this is resolved? 😆

Reply:
Thank you for flagging it, I fixed it somewhere else and forgot to push the one you commented: https://github.com/filecoin-project/go-f3/pull/259/commits/23a55118c16eb19fec6f76da605e352112748495#diff-4fa4e1c1a7b22e16444c976a3fecfaf2d47f9c48ca7d1ebcb955034e9daab511L148